doc/mpl_toolkits/index.rst

```diff
@@ -172,6 +172,26 @@ level interface for drawing statistical graphics with matplotlib. It
 aims to make visualization a central part of exploring and
 understanding complex datasets.
 
+.. image:: /_static/seaborn.png
+    :height: 157px
+
+.. _toolkit_holoviews:
+
+holoviews
+=========
+(*Not distributed with matplotlib*)
+
+`holoviews <http://holoviews.org>`_ makes it easier to visualize data
```

Comment:
A single `_` is the simplest link. but if you tried to use the word 'holoviews' for a different link elsewhere on the same page sphinx would complain as it was now confused where 'holoviews' should point.  Using `__` makes it an 'anonymous' link so you can re-use the same text with a different target elsewhere on the page. (as explained by @jenshnielsen )